src/components/blog-creator/BlogCreator.tsx

```diff
@@ -43,6 +43,25 @@ const CodeContainer = styled.div`
 
   .ui-editable-snippet {
     overflow: unset !important;
+    pre {
+      padding-left: 60px !important;
+    }
+    textarea {
+      outline: none;
+      padding-left: 60px !important;
+    }   
+    .editorLineNumber {
```

Comment:
use convention editor-line-number instead of editorLineNumber.

Reply:
Ok, done! :) 

[8b7537e](https://github.com/polubis/WebBlog/pull/232/commits/8b7537e8b3f3676debad5eea622c803e2ed0a6c4)